projects/make-a-vis/src/themes/shared/base.scss

```diff
@@ -121,6 +121,16 @@
     border-color: $border;
     background-color: $white;
   }
+  .about-expansion-panel {
+    border-color: $border !important;
+  }
+  .about-expansion-panel-header {
+    height: 2.2rem !important;
```

Comment:
height should be in the component, not in the theme

Reply:
removed everything from base.scss except theming stuff, like colors